<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGoodItem</class>
 <widget class="QMainWindow" name="AddGoodItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>584</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>新建/修改版图信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 5px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
}

QPushButton{
	background: #1890ff;
	border: 0px;
	border-radius: 3px;
	color: #fff;
}

QPushButton:hover{
	background: rgb(64,169,255);
}</string>
   </property>
   <widget class="QLabel" name="actionLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新建版图</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>版图 ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gidEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>版图编号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gnoEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>规格</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="skuEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>版图名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>描述</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subtitleEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>分类</string>
    </property>
   </widget>
   <widget class="QComboBox" name="categoryComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 5px;
    padding: 1px 2px 1px 2px; 
    min-width: 9em; 
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
 
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
 
QComboBox::down-arrow {
    image: url(:/dropdown.png);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>库存量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inventoryEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>140</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>进货价格</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="restockValueEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="sellingValueEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>出售价格</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lengthEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>长度 (mm)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>宽度 (mm)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>高度 (mm)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorEdit">
    <property name="geometry">
     <rect>
      <x>668</x>
      <y>180</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>颜色</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>商品展示图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadImageButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#uploadImageButton {
	color:	#3CB371;
	background:	white;
 	border:2px solid	#3CB371;
	border-radius: 3px;
}

#uploadImageButton:hover {
	background:		#3CB371;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>上传图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteImageButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>500</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#deleteImageButton {
	color:#FF4500;
	background:	white;
 	border:2px solid #FF4500;
	border-radius: 3px;
}

#deleteImageButton:hover {
	background:	#FF4500;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>删除图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>540</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>540</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#cancelButton {
	color:	#778899;
	background:	white;
 	border:2px solid 	#778899;
	border-radius: 3px;
}

#cancelButton:hover {
	background:		#778899;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>重量 (g)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weightEdit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableView" name="imageView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>841</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
